pyrightconfig.stricter.json

```diff
@@ -73,6 +73,7 @@
         "stubs/ttkthemes",
         "stubs/urllib3",
         "stubs/vobject",
+        "stubs/zstd",
```

Comment:
Stubs should be added here if they *fail* the stricter checks. I think the current stubs will pass the stricter checks, so we should remove this line.
```suggestion
```

Reply:
It does indeed. If it is so, maybe the `create_baseline_stubs.py` script should be adapted to make it clearer. By default, it adds this line. Maybe it would be relevant at least to point out in the output this line should be deleted manually when the user finished editing the stubs.